engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelHydrationValidation.kt

```diff
@@ -157,7 +157,23 @@ internal class NadelHydrationValidation(
             }
         }
 
-        return duplicatedArgumentsErrors + remoteArgErrors
+        val missingActorArgErrors = actorField.arguments.mapNotNull { actorArg ->
+            val hydrationArg = hydration.arguments.find { it.name.equals(actorArg.name) }
+            if (hydrationArg == null) {
+                //this is allowed if the actor field arg is nullable so check for that
```

Comment:
If you're looking at a functional way to do it, you should add a filter on `arguments` and filter only the arguments with non-nullable types